command_test.go

```diff
@@ -315,5 +315,6 @@ func TestCommandSkipFlagParsing(t *testing.T) {
 		err := app.Run(c.testArgs)
 		expect(t, err, c.expectedErr)
 		expect(t, args, c.expectedArgs)
+		expect(t, value, "")
```

Comment:
Looking at some of the other PRs that have started failing recently, this line is needed to fix a `go vet` failure that was likely introduced with [an update to `vet` in Go 1.10](https://golang.org/doc/go1.10#vet).